code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -615,19 +615,20 @@
 			return
 
 		if(QUALITY_SCREW_DRIVING)
-			if(opened && !cell)
+			if (opened && !cell)
 				if(I.use_tool(user, src, WORKTIME_FAST, tool_type, FAILCHANCE_NORMAL, required_stat = STAT_MEC))
 					wiresexposed = !wiresexposed
 					user << SPAN_NOTICE("The wires have been [wiresexposed ? "exposed" : "unexposed"]")
 					updateicon()
-					return
-			if(opened && cell)
+			else if (module_active && istool(module_active))
```

Comment:
likewise here, present a list of all tools with installed upgrades to choose from